snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsTypes/CS/keywordsTypes.cs

```diff
@@ -98,46 +98,39 @@ int y
           get;
           set;
        }
+
+       public double Distance 
+       {
+           get;
+       }
     }
 
     class Point : IPoint
```

Comment:
The changes requesting to rename the properties from lowercase to uppercase should be done in the interface too.